features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ExploreViewModel.kt

```diff
@@ -22,6 +22,7 @@ import androidx.annotation.VisibleForTesting
 import androidx.lifecycle.*
 import androidx.paging.*
 import androidx.paging.PagingData
+import com.google.android.gms.maps.model.LatLng
```

Comment:
The viewModel should not reference LatLng since it's implementation-dependent and we're working with an interface in case of `UserLocationState`. If the implementation of `UserLocationState` changed, you'll have to change a lot of related code in the viewModel as well. Please use `GeoBounds` instead.